File: exercism/javascript/etl/etl.js
const { forEach } = require('lodash/fp')

class ETL {
  transform (given) {
    const result = {}
    const addScores = (letters, score) =>
      letters.forEach(l => result[l.toLowerCase()] = parseInt(score))
    forEach(addScores, given)
    return result
  }
}

module.exports = ETL

File: exercism/go/etl/etl.go
package etl

// Transform returns a transformed scrabble score format
func Transform(given map[int][]string) map[string]int {
	out := make(map[string]int)
	for k, v := range given {
		for _, c := range v {
			out[string(c[0]+32)] = k
		}
	}
	return out
}
